package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "screwdriver-scm-bitbucket",
-  "version": "2.3.0",
+  "version": "4.0.0",
```

Comment:
why not 3 ?

Reply:
it's already at 3 if you check the git tags

Reply:
Current versions is https://www.npmjs.com/package/screwdriver-scm-bitbucket. 3.x.x